runners/sbt/src/main/scala/stryker4s/sbt/SbtMutantRunner.scala

```diff
@@ -33,8 +34,7 @@ class SbtMutantRunner(state: State, processRunner: ProcessRunner, sourceCollecto
 
   private val settings: Seq[Def.Setting[_]] = Seq(
     fork in Test := true,
-    scalaSource in Compile := tmpDirFor(Compile).value,
-    scalaSource in Test := tmpDirFor(Test).value,
+    scalaSource in Compile := tmpDirFor(Compile).value
   ) ++
```

Comment:
`scalaSource in Test` is no longer necessary?

Reply:
We don't mutate test sources, so there's no point in setting them to the new tmpDir

Reply:
Right, good catch 😅